.env.docker.example

```diff
@@ -1,98 +0,0 @@
-APP_ENV=local
-APP_KEY=SomeKey
-APP_URL=http://localhost:8080/
-FALLBACK_COOKIE_DOMAIN=localhost
-
-DEBUG=true
-
-SSL=false
-
-DB_HOST=mariadb
-DB_PORT=3306
-DB_DATABASE=saproto
-DB_USERNAME=saproto
-DB_PASSWORD=saproto
-EMAIL_DB_DATABASE=email_database_name
-
-CACHE_DRIVER=file
-SESSION_DRIVER=database
-QUEUE_DRIVER=database
-
-BROADCAST_DRIVER=log
-
-MAIL_DRIVER=smtp
-SMTP_HOST=mailhog
-SMTP_PORT=1025
-SMTP_USERNAME=user
-SMTP_PASSWORD=password
-
-GOOGLE_KEY_PUBLIC=API_KEY
-GOOGLE_KEY_PRIVATE=API_KEY
-
-RECAPTCHA_PUBLIC_KEY=PUBLIC_KEY
-RECAPTCHA_PRIVATE_KEY=PRIVATE_KEY
-
-FLICKR_USER=141430890@N05
-FLICKR_CLIENT=FlickrKey
-FLICKR_SECRET=FlickrToken
-
-SPOTIFY_USER=studyassociationproto
-SPOTIFY_CLIENT=SpotifyKey
-SPOTIFY_SECRET=SpotifySecret
-SPOTIFY_PLAYLIST=PlayListId
-
-SENTRY_DSN=https://xyz:xyz@sentry.io/123
-SENTRY_PUBLIC_DNS=https://foo:bar@sentry.io/123
-
-PRINTER_HOST=printer.proto.utwente.nl
-PRINTER_PORT=12345
-PRINTER_SECRET=SomeKey
-
-HERBERT_SERVER=https://proto.utwente.nl:1234
-HERBERT_SECRET=SECRET_KEY
-
-FISHCAM_URL=https://proto.utwente.nl/api/fishcam
-
-LDAP_PROXY_URL=https://path.to/proxy
-LDAP_PROXY_KEY=VERY_SECRET_KEY
-
-MOLLIE_TEST_MODE=true
-MOLLIE_KEY_LIVE=live_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-MOLLIE_KEY_TEST=test_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
-
-SLACK_TOKEN=slack_token
-
-SAML2_SP_CERT=X509_CERT_DATA
-SAML2_SP_KEY=X509_KEY_DATA
-
-SAML2_SP_REMOTE_ENTITY_ID=https://idp.provider.tld
-SAML2_SP_REMOTE_SOS=https://sos.url
-SAML2_SP_REMOTE_SLS=https://sls.url
-SAML2_SP_REMOTE_CERT=X509_CERT_DATA
-
-SAML2_ATTR_FIRSTNAME=urn:mace:dir:attribute-def:givenName
-SAML2_ATTR_SURNAME=urn:mace:dir:attribute-def:sn
-SAML2_ATTR_EMAIL=urn:mace:dir:attribute-def:mail
-SAML2_ATTR_UID=urn:mace:dir:attribute-def:uid
-
-SAML2_IDP_ENTITY_ID=https://idp.proto.utwente.nl
-SAML2_IDP_CERT=X509_CERT_RELATIVE_PATH
-SAML2_IDP_KEY=X509_KEY_RELATIVE_PATH
-
-PERSONAL_PROTO_KEY=personal_proto_key
-
-HASHIDS_SECRET_EVENTS=RAND_STRING
-
-SEPA_IBAN=IBAN
-SEPA_BIC=BIC
-SEPA_CI=CREDITOR_ID
-
-PRIMARY_DOMAIN=http://localhost:8080
-REDIRECT_ASSETS_DOMAIN=https://www.proto.utwente.nl
-
-DOCKER_HTTP_PORT=8080
-DOCKER_PMA_PORT=8081
-DOCKER_MAILHOG_PORT=8082
-DOCKER_SQL_PORT=3306
-
-ANALYTICS_URL=analytics.saproto.nl
```

Comment:
Please do not delete this file, it a nice friendly file that deserves love

Reply:
My past self would have not agreed.